config/samples/controlplane.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       spec:
         containers:
         - name: controller
-          image: kong/kubernetes-ingress-controller:3.1
+          image: kong/kubernetes-ingress-controller:3.1.4
```

Comment:
This all depends on the `imagePullPolicy` (which we do not set) and the [defaulting behavior](https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting).

My take on this is that we'd want to add renovate code markers here so that we do not need to bump those anymore. WDYT?

Reply:
Yep, it'd be great to have these automatically bumped 👍